### DISCUSSION TITLE
Does PNP work with SPSE?

### DISCUSSION THREAD

**Discussion by mikelee1313** [pnp/powershell#3042] [General]
Now that SharePoint Server Subscription Edition has been released, has PNP been tested to work with SPSE?

**Comment by markgort86** [comment]
This question is also answered over here: https://github.com/pnp/powershell/discussions/3147. You can still use SharePoint 2019 PnP but this module is not updated anymore. You can download this version by using the following hyperlink: https://www.powershellgallery.com/packages/SharePointPnPPowerShell2019/3.29.2101.0
